Dockerfile

```diff
@@ -12,10 +12,32 @@ RUN curl -sL https://deb.nodesource.com/setup_6.x | bash \
     && apt-get install --no-install-recommends -y nodejs \
     && rm -rf /var/lib/apt/lists/*
 
+# Kosmtik with plugins
 RUN npm install -g kosmtik
 
+WORKDIR /usr/lib/node_modules/kosmtik/
```

Comment:
Something seems to be wrong with plugins npm installation. When Kosmtik is installed globally and plugins are installed with "i" - as advised on their npm pages - plugins are installed in the local (working) directory and are not visible to the program. I was trying to find a way to install them in `/usr/lib/node_modules/kosmtik/node_modules/` through npm options (like `--prefix`), but failed.

I guess npm installation of plugins needs to be examined by the Kosmtik team to work out of the box.